pkg/config/capabilities.go

```diff
@@ -57,7 +57,153 @@ func (c *ConfigTx) GetApplicationCapabilities() (map[string]bool, error) {
 	return capabilities, nil
 }
 
-func getCapabilities(configGroup *cb.ConfigGroup) (map[string]bool, error) {
+// AddChannelCapability adds capability to the provided channel config,
+// capability must be specified.
```

Comment:
Regarding this comment, where does it enforce the fact that the capability being added must be specified.

Reply:
It doesn't enforce it, only expected. I can remove this from the doc.